wordguess.rb

```diff
@@ -0,0 +1,109 @@
+# Emilce C  & Maria Velez
+# Ada C9 - Octos - Word-Guess
+# 12th - 14th Feb 2018
+# https://github.com/Ada-C9/Word-Guess
+
+class Art
+  attr_accessor :art
+
+  def initialize
+    @art = art
```

Comment:
Here you have an instance variable `@art`, but I don't see that it's ever used. You're also setting a value for it, but your constructor does not take a `art` parameter.